packages/extras/src/lib/components/controls/TransformControls/TransformControls.svelte

```diff
@@ -22,14 +22,17 @@
   type $$Slots = TransformControlsSlots
 
   export let autoPauseOrbitControls: $$Props['autoPauseOrbitControls'] = true
+  export let autoPauseTrackballControls: $$Props['autoPauseTrackballControls'] = true
   export let object: $$Props['object'] = undefined
 
   const { camera, renderer, invalidate, scene } = useThrelte()
 
-  const { orbitControls } = useControlsContext()
+  const { orbitControls, trackballControls } = useControlsContext()
   const isDragging = writable(false)
   const useAutoPauseOrbitControls = writable(autoPauseOrbitControls) ?? true
   $: useAutoPauseOrbitControls.set(autoPauseOrbitControls ?? true)
+  const useAutoPauseTrackballControls = writable(autoPauseTrackballControls) ?? true
```

Comment:
```suggestion
  const useAutoPauseTrackballControls = writable(autoPauseTrackballControls ?? true)
```
Boolean was accidently placed outside of the store args

Reply:
whoa TIL that github has a suggested changes feature!

Reply:
It's such a great feature

Reply:
fixed!